p256k1/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "p256k1"
-version = "4.0.1"
+version = "4.1.0"
```

Comment:
If you're swapping deps and/or doing anything to pub types other than adding fields/funcs, it's a major version bump